apps/src/sites/studio/pages/schoolInfo.js

```diff
@@ -6,6 +6,17 @@ window.SchoolInfoManager = function (existingOptions) {
 
   var districtElement = $('#school-district-id');
 
+  // Comparison function to sort objects by their 'name' property.
+  function byName(entry1, entry2) {
+    if (entry1.name < entry2.name) {
+      return -1;
+    }
+    if (entry1.name > entry2.name) {
+      return 1;
+    }
+    return 0;
+  }
+
```

Comment:
Nice catch. Your reading is correct, `sortField` works great. I didn't see it because I was looking at the [api docs](https://github.com/selectize/selectize.js/blob/master/docs/api.md) which misleadingly doesn't document much of the api.